README.md

```diff
@@ -83,7 +241,13 @@ export LIBFABRIC_DIR="where/you/installed/your/libfabric"
 export LD_LIBRARY_PATH="$LIBFABRIC_DIR/lib:$MERCURY_DIR/lib:$LD_LIBRARY_PATH"
 export PATH="$PDC_DIR/include:$PDC_DIR/lib:LIBFABRIC_DIR/include:$LIBFABRIC_DIR/lib:$MERCURY_DIR/include:$MERCURY_DIR/lib:$PATH"
 ```
+
+-->
+
+## About Spack
+
 One can also manage the path with Spack, which is a lot more easier to load and unload these libraries.
```

Comment:
"lot more easier" --> "lot easier"

Reply:
resolved. 

Also, the paragraphs after this line got revised.